gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1326,26 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+void
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env, MM_HeapWalkerObjectFunc walkFunction)
+{
+	HeapSizes counter;
+	counter.freeBytes = 0;
+	counter.objectBytes = 0;
```

Comment:
does the below work? there should be an easier/cleaner way to initialize.
```suggestion
	HeapSizes counter = {0, 0};
```